index.js

```diff
@@ -142,7 +240,26 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
+  if(movies.length === 0){
+    return null
+  }
+
+  let highestAmount = 0
+  let name;
+
+  for (movie of movies){
+    let boxOfficeAmount = movie.boxOffice.slice(1)
+    let boxOfficeAmountNoCommas = boxOfficeAmount.replace(/,/g, '');
+    let num = Number(boxOfficeAmountNoCommas)
+    if (num > highestAmount){
+      highestAmount = num
+      name = movie.title
+    }
+  }
+
+  return name
```

Comment:
Nice work with the `regex`.  Could you add a `$` in there to completely sanitize this text?